docs/data/material/components/pickers/deprecated-pickers.md

```diff
@@ -0,0 +1,19 @@
+# Date / Time pickers
+
+<p class="description">Now part of MUI X, a collection of advanced components.</p>
```

Comment:
Let's make this a complete sentence, and name the components that have been moved. Since it looks like there are quite a few, it may be best to just give some examples at least.

```suggestion
<p class="description">The `<DatePicker>`, `<TimePicker>`, and related components have been moved to MUI X—our collection of advanced components.</p>
```